src/Symfony/Component/Translation/CatalogueBag/ContainerAwareResourceCatalogueBag.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Translation\CatalogueBag;
```

Comment:
not sure, the  [EventDispatcher](https://github.com/symfony/symfony/blob/2.8/src/Symfony/Component/EventDispatcher/ContainerAwareEventDispatcher.php) contains similar class.